mandatory/2-function-creation.js

```diff
@@ -4,37 +4,84 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
-function isAcceptableUser(userAge, isLoggedIn) {}
+function isAcceptableUser(userAge, isLoggedIn) {
+  return userAge >= 18 && isLoggedIn;
+}
+
+const loggedIn = true;
+const userAge = 21;
+
+if (isAcceptableUser(userAge, loggedIn)) {
+  console.log("User is acceptable.");
+} else {
+  console.log("User is not acceptable.");
+}
 
 /*
   Complete the function to apply discount percent based on how much is totalPrice in user cart.
     - When the total price is greater than 200, a 10% discount should be applied
     - When the total price is less than 200, a 5% discount should be applied
 
   The function should return the new price to be paid (e.g. if the totalPrice is 150, a 5% discount
-    is applieds and 142.5 should be returned)
+    is applied and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  let discountPercent;
+  if(totalPrice > 200) {
+    discountPercent = 0.1;
+  } else {
+    discountPercent = 0.05;
+  }
+  let discountAmount = totalPrice * discountPercent;
+  let newPrice = totalPrice - discountAmount;
+  return newPrice;
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  let i = 1;
+  while (i <= limit) {
+    if(i % 2 !==0) {
+      console.log(i);
+    }
+    i++;
+}
+}
```

Comment:
this works fine, but you could make it 2x more performant by skipping over even numbers. For example:

```suggestion
function printOddNumbers(limit) {
  let i = 1;
  while (i <= limit) {
    console.log(i);
    i+=2;
}
}
```